hmac_authentication/tests/test_hmac_authentication.py

```diff
@@ -129,45 +111,47 @@ def setUp(self):
             'PATH_INFO': '/foo/bar?baz=quux%2Fxyzzy#plugh',
         }
 
-    def test_validate_request_no_signature(self):
-        result, header, computed = self.auth.validate_request(self.environ)
-        self.assertEqual(HmacAuth.NO_SIGNATURE, result)
+    def test_authenticate_request_no_signature(self):
+        result, header, computed = self.auth.authenticate_request(self.environ)
+        self.assertEqual(AuthenticationResultCodes.NO_SIGNATURE, result)
```

Comment:
OK, I'll take a stab at pytest. Just never used it before. (And looking at the Testing Cookbook, it might be nice for us to start splitting parts of it up into child pages.)